colors.txt

```diff
@@ -0,0 +1 @@
+#0f1078
```

Comment:
whar

if this is
1. a color pallete: we should have those defined as variables in a `:root` element in some css file
2. some weird temporary file: get rid of it
3. something else: explain what the hell it is cuz i can't figure it out